homeassistant/components/sensor/iss.py

```diff
@@ -0,0 +1,125 @@
+"""
+Support for International Space Station data sensor.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.iss/
+"""
+import logging
+from datetime import timedelta, datetime
+import requests
+import voluptuous as vol
+from homeassistant.util import Throttle
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (CONF_NAME)
+from homeassistant.helpers.entity import Entity
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['pyiss==1.0']
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_ISS_VISIBLE = 'visible'
+CONF_ISS_NEXT_RISE = 'next_rise'
+CONF_TIME_SECOND = 'mn'
+
+DEFAULT_NAME = 'Iss'
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=60)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+})
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup the ISS sensor."""
+    # Validate the configuration
+    if None in (hass.config.latitude, hass.config.longitude):
+        _LOGGER.error("Latitude or longitude not set in Home Assistant config")
+        return False
+
+    try:
+        iss_data = IssData(hass.config.latitude, hass.config.longitude)
+        iss_data.update()
+    except requests.exceptions.HTTPError as error:
+        _LOGGER.error(error)
+        return False
+
+    name = config.get(CONF_NAME)
+
+    sensors = []
+    sensors.append(IssSensor(iss_data, name, CONF_ISS_VISIBLE))
+    sensors.append(IssSensor(iss_data, name, CONF_ISS_NEXT_RISE))
```

Comment:
You mean have only one sensor which tell you is ISS is visible or not and next rise and number of people in space as state attributes ? It seems good and logical...

By the way thanks for PR for the pyiss library.